requirements/dev-requirements.in

```diff
@@ -0,0 +1,22 @@
+django-debug-toolbar==1.8
+line_profiler==1.0
+-e git+git://github.com/dimagi/couchdbkit-debugpanel.git#egg=couchdbkit-debugpanel
+Sphinx==1.3.1
+sphinx-rtd-theme==0.1.9
+jenkinsapi==0.2.28
+fixture==1.5.11
+sniffer==0.4.0
+psutil==5.1.3  # for memory profiling
+django-extensions==1.7.7
+ipython==5.2.2
+ipdb==0.11
+wheel==0.29.0
+graphviz==0.7
+-e git+git://github.com/nickpell/django-package-monitor@np/handle-no-semver#egg=django-package-monitor
+transifex-client==0.12.4
+modernize==0.6
+Fabric==1.14.0
+readline==6.2.4.1
+pip-tools==2.0.2
+
+--requirement requirements.txt
```

Comment:
is there any difference between having this at the top of the file or the bottom? I notice it's at the bottom here and at the top of `test-requirements.in` and `docs-requirements.in`